JDI.Light/JDI.Light.Tests/Tests/Simple/CheckBoxTests.cs

```diff
@@ -23,7 +23,12 @@ public void SetUp()
         [Test]
         public void CheckSingleTest()
         {
-            Assert.DoesNotThrow(() => TestSite.MetalsColorsPage.CbWater.Check(true));
+            /*
+            5/19/2020: update Alex T remove true
+            from line30: CbWater.Check(true));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5552759)